### PR TITLE
Update to nushell 0.59.1

### PR DESCRIPTION
**Pull request by chvck**

### CODE REVIEW

File: Cargo.toml
Comment (chvck) [inline Cargo.toml:29]: These will obviously need to change, once my changes are merged upstream.

File: src/cli/cbenv_bucket.rs
Comment (chvck) [inline src/cli/cbenv_bucket.rs:23]: Open to better names than cb-env but given other nushell commands I think it makes sense.

File: Cargo.toml
Comment (ingenthr) [reply Cargo.toml]: I think that's done now, right?  Need to update this PR?

Comment (chvck) [reply Cargo.toml]: The PR has been updated already, it was using local paths but is now using a git SHA. Nushell 0.60 is out now so once this PR is in we can make a new patchset to bump to 0.60 and use a proper tagged version again.

File: src/cli/cbenv_cmd.rs
Comment (ingenthr) [inline src/cli/cbenv_cmd.rs:153]: similar comment here.  seems like the value should be some kind of optional and what you have here is really formatting.

File: src/cli/cbenv_project.rs
Comment (ingenthr) [inline src/cli/cbenv_project.rs:51]: was this intentionally different?

**Review by ingenthr — APPROVED** [review]
I'm going to do some additional looking around here and possibly change my review.

I have some comments that might be more improvements.  I don't think they should block merge, but they may just be things we can improve in the configuration type system so to speak.

File: src/cli/cbenv_bucket.rs
Comment (chvck) [reply src/cli/cbenv_bucket.rs:23]: >  Are there popular extensions

No :)

File: docs/cloud.adoc
Comment (chvck) [reply docs/cloud.adoc:12]: `use` is now a nushell language keyword. It's not even a command in the way that something like `ls` is or we could make a workaround for it.

File: src/cli/buckets_builder.rs
Comment (chvck) [reply src/cli/buckets_builder.rs:46]: We definitely need to go through and improve errors in general to improve UX, there is an issue open for that.

File: src/cli/cbenv_bucket.rs
Comment (chvck) [reply src/cli/cbenv_bucket.rs:58]: This is just the name to display within the table, it's not actually ever used in code as such.

File: src/cli/cbenv_project.rs
Comment (chvck) [reply src/cli/cbenv_project.rs:51]: An oversight I think.

**Pull request merged**
